utils/tfhe-versionable/README.md

```diff
@@ -1,9 +1,14 @@
 # TFHE-versionable
-This crate provides type level versioning for serialized data. It offers a way to add backward
-compatibility on any data type. The versioning scheme works recursively and is independant of the
-chosen serialization backend.
+This crate provides type level versioning for serialized data. It offers a way
+to add backward compatibility on any data type. The versioning scheme works
+recursively and is independant of the chosen serialization backend.
```

Comment:
Maybe worth indicating it uses the serde framework, even though it's almost a given in rust